highLevelInterface/pwaFit.cc

```diff
@@ -63,8 +63,7 @@ namespace ROOT {
 
 fitResultPtr
 rpwa::hli::pwaFit(const pwaLikelihood<complex<double> >& L,
-                  const double                           massBinMin,
-                  const double                           massBinMax,
+                  const binningMapType&                  binningMap,
```

Comment:
This could take an empty map as a default argument, you in any case do this for the Python interface.